api/auth/index.ts

```diff
@@ -107,6 +107,14 @@ async function sessionChainCheck(req: EnhancedRequest, res: express.Response, ac
     return true
 }
 
+export function havePrdAdminRole(userData){
```

Comment:
Changing this function to `export function havePrdAdminRole(userData){
	return userData.data.roles.indexOf('prd-admin') === -1;
}` 

Will give the same result from this function, thereby removing the need for returning false / true.